protocol/communities/manager.go

```diff
@@ -1209,7 +1209,7 @@ func (m *Manager) AddMemberToCommunity(communityID types.HexBytes, pk *ecdsa.Pub
 		return nil, ErrOrgNotFound
 	}
 
-	err = community.AddMember(pk)
+	err = community.AddMember(pk, []protobuf.CommunityMember_Roles{protobuf.CommunityMember_ROLE_ALL})
```

Comment:
Could be wrong but should admins really get `ROLE_ALL`? Thought `ROLE_MANAGE_USERS` would be more appropriate.

Reply:
Admin != owner but I think you pointed that out here ` Now, I think that maybe Owner (instead of Admin) will be more correct word, so probably I will rename it.`